css/index.css

```diff
@@ -105,18 +105,17 @@ a{
 	border: black;
 
 }
-
 .button{
 	display: flex;
 	justify-content: center;
-	margin-left: 60px;
-	padding: 5px 35px 5px 35px;
+	margin-left: 22%;
+	padding: .9% 10% .9% 10%;
```

Comment:
It's more common to see top and bottom to be in px, and left and right to be in percentages.